trillian/ctfe/handlers_test.go

```diff
@@ -340,6 +413,13 @@ func TestAddChain(t *testing.T) {
 	defer info.mockCtrl.Finish()
 
 	for _, test := range tests {
+		if test.enableRemoteQuota {
+			// first entry in wantQuotaUsers is the remote user
+			info.setRemoteQuotaUser(test.wantQuotaUsers[0])
```

Comment:
This is a bit hard to follow (pulling values from the expected test output to serve as injected inputs) -- maybe just use a hard-coded remote-user string and shift `setRemoteQuotaUser()` to take a boolean?

Reply:
changed